pkg/gadgets/snapshot/socket/tracer/tracer.go

```diff
@@ -161,14 +166,15 @@ func RunCollector(pid uint32, podname, namespace, node string, proto socketcolle
 					return err
 				}
 
+				// TODO: Receive the netns from caller
+				netns, err := containerutils.GetNetNs(int(pid))
+				if err != nil {
+					return fmt.Errorf("getting netns for pid %d: %w", pid, err)
+				}
+
 				sockets = append(sockets, &socketcollectortypes.Event{
 					Event: eventtypes.Event{
 						Type: eventtypes.NORMAL,
-						CommonData: eventtypes.CommonData{
-							Node:      node,
-							Namespace: namespace,
-							Pod:       podname,
-						},
```

Comment:
This is causing the issue with the CI. It can't be removed until we start using `Tracer.Run` method by kubectl-gadget. We can add a comment to not forget it as I did in my proposal: https://github.com/inspektor-gadget/inspektor-gadget/commit/9272221c06a3cfc058a60ee3a1e325d7ac9cd20b#diff-cf9f0e218778da1c45350fbdb0053479f23d66a86800100810a98a44d0fb8dd2